diff -Naur mozplugger-1.14.6.orig/mozpluggerrc mozplugger-1.14.6/mozpluggerrc
--- mozplugger-1.14.6.orig/mozpluggerrc	2012-12-04 08:26:06.461813768 -0500
+++ mozplugger-1.14.6/mozpluggerrc	2012-12-04 08:26:13.400813169 -0500
@@ -28,7 +28,7 @@
 
 ### MPlayer
 
-define(MP_CMD,[mplayer -really-quiet -nojoystick -nofs -zoom -vo xv,x11 -ao esd,alsa,oss,arts,null -osdlevel 0 $1 </dev/null])
+define(MP_CMD,[mplayer -really-quiet -nojoystick -nofs -zoom -vo xv,x11 -ao pulse,esd,alsa,oss,arts,null -osdlevel 0 $1 </dev/null])
 
 define(MP_EMBED,[embed noisy ignore_errors: MP_CMD(-xy $width -wid $window $1)])
 
